cli/cli/main.go

```diff
@@ -49,11 +46,17 @@ func main() {
 	})
 
 	if err := commands.RootCmd.Execute(); err != nil {
-		maybeCleanedError := getErrorMessageToBeDisplayedOnCli(err)
+		maybeCleanedError := out.GetErrorMessageToBeDisplayedOnCli(err)
 		errorMessageFromCli := maybeCleanedError.Error()
-		// cobra uses this method underneath as well to print errors
-		// so just used it directly.
-		commands.RootCmd.PrintErrln(errorPrefix, errorMessageFromCli)
+
+		// This error thrown by the APIC is not informative right now as it just tells the user to look at errors
+		// in the above log. For this reason we're ignoring it and returning nil. This is exceptional to not clutter
+		// the CLI output. We should still use stacktrace.Propagate for other errors.
+		if !strings.Contains(errorMessageFromCli, command_str_consts.ErrorMessageDueToStarlarkFailure) {
```

Comment:
you could use `errors.Is(errorMessageFromCli, ErrorMessageDueToStarlarkFailure)` here instead, if `ErrorMessageDueToStarlarkFailure` is an error object.

Reply:
Let me try this - because we are using propagate wouldn't this be false - but let me try

Reply:
oh! yes that is right, you could use `stacktrace.RootCause()` to get the root cause error, before you will have to cast the error, but yes probably is too much, let it as it is now.

Reply:
hehe already did it; no problem. @leoporoli